docs/usermanual.md

```diff
@@ -9,7 +9,7 @@
 * [Development documentation](#development)
 * [New transport development](#newtransport)
 
-## <a name="apioverview"/> API Overview
+## API Overview
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated